Makefile

```diff
@@ -57,6 +61,77 @@ ifeq (SIGN, 1)
 endif
 	@echo "ZREPL RELEASE ARTIFACTS AVAILABLE IN artifacts/release"
 
+rpm: $(ARTIFACTDIR) # artifacts/_zrepl.zsh_completion artifacts/bash_completion docs zrepl-bin
+ifneq (,$(filter $(HOST_OS_ID),fedora centos rhel))
+	$(info Generating rpm package requires the rpm build utilities to\
+	       be installed. To install the minimal set of required packages:)
+	$(info )
+	$(info dnf install rpm-build rpmdevtools)
+	$(info )
+else
+	$(error Building rpm packages has only been tested on Fedora and \
+	        is expected to work on RedHat/CentOS >= 8. Current host OS is:\
+	        $(HOST_OS_NAME))
+endif
+	$(eval _ZREPL_RPM_VERSION := $(subst -,.,$(_ZREPL_VERSION)))
+	$(eval _ZREPL_RPM_TOPDIR_ABS := $(CURDIR)/$(ARTIFACTDIR)/rpmbuild)
+	rm -rf "$(_ZREPL_RPM_TOPDIR_ABS)"
+	mkdir "$(_ZREPL_RPM_TOPDIR_ABS)"
+	mkdir -p "$(_ZREPL_RPM_TOPDIR_ABS)"/{SPECS,RPMS,BUILD,BUILDROOT}
+	sed "s/^Version:.*/Version:          $(_ZREPL_RPM_VERSION)/g" \
+	    packaging/rpm/zrepl.spec >  $(_ZREPL_RPM_TOPDIR_ABS)/SPECS/zrepl.spec
+
+	# see /usr/lib/rpm/platform
+ifeq ($(GOARCH),amd64)
+	$(eval _ZREPL_RPMBUILD_TARGET := x86_64)
+else ifeq ($(GOARCH), 386)
+	$(eval _ZREPL_RPMBUILD_TARGET := i386)
+else ifeq ($(GOARCH), arm64)
+	$(eval _ZREPL_RPMBUILD_TARGET := aarch64)
+else
+	$(eval _ZREPL_RPMBUILD_TARGET := $(GOARCH))
+endif
+	rpmbuild \
+		--build-in-place \
+	  --define "_sourcedir $(CURDIR)" \
+	  --define "_topdir $(_ZREPL_RPM_TOPDIR_ABS)" \
+	  --define "_zrepl_binary_filename zrepl-$(ZREPL_TARGET_TUPLE)" \
+	  --target $(_ZREPL_RPMBUILD_TARGET) \
+	  -bb "$(_ZREPL_RPM_TOPDIR_ABS)"/SPECS/zrepl.spec
+	cp "$(_ZREPL_RPM_TOPDIR_ABS)"/RPMS/*/zrepl-$(_ZREPL_RPM_VERSION)*.rpm $(ARTIFACTDIR)/
```

Comment:
```suggestion
	cp "$(_ZREPL_RPM_TOPDIR_ABS)"/RPMS/$(_ZREPL_RPMBUILD_TARGET)/zrepl-$(_ZREPL_RPM_VERSION)*.rpm $(ARTIFACTDIR)/
```

Reply:
Why are you proposing this? The fedora 32 container separates creates one subdirectory per architecture.

Reply:
Because we know the value of this for sure now and adding it explicitly will help to prevent unforeseen issues due to a unnecessary wildcard - simply a safe-by-design choice.